pegasus/sites.v3/code.org/public/maker/index.haml

```diff
@@ -3,118 +3,102 @@ title: Become a Maker | Learn about Physical Computing
 
 %section
   .wrapper
-    .text-wrapper.col-60
-      %h2 What is a Maker?
-      %p <strong>Makers invent, explore, innovate, and create using technology like development boards, robotics, 3D printing, programming, and more.</strong>
-      %p.body-three Maker education takes STEM (Science, Technology, Engineering, Mathematics) a step further by inviting hands-on experience and collaboration into the classroom to help teach problem solving and critical thinking.
-    %div.col-33{style: "float: right"}
-      %img{src: "/images/maker/maker-students-cp.png", alt: "", style: "width: 100%; margin-top: .75em"}
-    .clear
+    .grid-container.two-col.justify-space-between.align-items-center.gap-4
+      %figure
+        %img.rounded-corners{src: "/images/maker/maker-students-cp.png", alt: "", style: "width: 100%"}
+      .text-wrapper
+        %h2=hoc_s("maker_page.what.heading")
+        %p.heading-xs{style: "margin-bottom: 1rem"}
+          =hoc_s("maker_page.what.desc_01")
+        %p=hoc_s("maker_page.what.desc_02")
 
+-# TODO - remove the divider line and unhide case study section when we get the link
 = view :section_divider_line
+-# %section.bg-neutral-dark
+-#   .wrapper.flex-container.justify-space-between.align-items-center.flex-direction-column-mobile.gap-2
+-#     .text-wrapper{style: "flex: 0.9"}
+-#       %p.overline-two.white
+-#         =hoc_s("maker_page.case_study.overline")
+-#       %h2.heading-lg.white
+-#         =hoc_s("maker_page.case_study.heading")
+-#       %p.body-three.white.no-margin-bottom
+-#         =hoc_s("maker_page.case_study.desc")
+-#     %img{src: "/images/maker/alaska.png", alt: "", style: "width: 182px"}
+-#     %a.link-button.white.has-external-link{href: "#", target: "_blank", rel: "noopener noreferrer"}
+-#       =hoc_s("maker_page.case_study.button")
 
-%section#explore_devices_link
+%section
   .wrapper
-    %h2 Explore devices
-    %p We have curriculum options that support two different devices — see available options below. <strong>Need a device?</strong> Learn more about the devices below.
-    = view :maker_devices, curriculum_link_cp: "#curriculum_link_cp", curriculum_link_microbit: "#curriculum_link_microbit",  button_label: "See available curriculum", aria_label_cp: "See available curriculum for Circuit Playground device", aria_label_microbit: "See available curriculum for BBC micro:bit device"
+    %h2=hoc_s("maker_page.get_started.heading")
+    %p=hoc_s("maker_page.get_started.desc")
+    = view :maker_devices, curriculum_link_cp: "#curriculum_link_cp", curriculum_link_microbit: "#curriculum_link_microbit",  button_label: hoc_s("maker_page.get_started.button.see_curriculum.label"), aria_label_cp: hoc_s("maker_page.get_started.button.see_curriculum.aria_label", markdown: :inline, locals:{device: "BBC micro:bit"}), aria_label_microbit: hoc_s("maker_page.get_started.button.see_curriculum.aria_label", markdown: :inline, locals:{device: "Circuit Playground"})
```

Comment:
If we're making the other strings in here translatable, should we also make the aria label of "Circuit Playground" translatable too?

Reply:
I think this is the same idea as [this comment](https://github.com/code-dot-org/code-dot-org/pull/59387/files#r1657902575) re: the names shouldn't be translated since they are proper nouns.